src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsWindowMapping.java

```diff
@@ -46,6 +53,10 @@ public static HearingWindow buildHearingWindow(@Valid SscsCaseData caseData) {
     }
 
     public static LocalDate getHearingWindowStart(@Valid SscsCaseData caseData) {
+        if (isCaseAdjourned(caseData)) {
+            return calculateAdjournedHearingDate(caseData);
+        }
+
         if (isCasePostponed(caseData)) {
```

Comment:
Suggestion: turn this into an if/else if/else

Reply:
done